common/v2/features/Toasts/ToastProvider.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component, createContext } from 'react';
+import toast from 'toasted-notes';
+
+import { default as ToastComponent } from './components/Toast';
+import { toastConfigs } from './constants';
+
+export interface ProviderState {
```

Comment:
Is there a reason to export the interface ?

Reply:
Don't think so - It's copied from `NotificationsProvider`, I can remove if not needed!

Reply:
Please do. Having a clean interface for the module simplifies maintainability.

Reply:
Already done 😄